packages/typegen/src/query/parse.ts

```diff
@@ -339,6 +339,7 @@ if (require.main === module) {
     '<<': 'binary_left_shift',
     '>>': 'binary_right_shift',
     '#': 'bitwise_xor',
+    'AT TIME ZONE': 'at_time_zone',
```

Comment:
Hah, this is embarrassing, I forgot all this madness is still here. This is a script which I’ve used for debugging but it’s accidentally shipped with the package!